functions/postSurveyInit.ts

```diff
@@ -36,7 +38,7 @@ const createSurveyTask = async (context: Context<EnvVars>, event: Required<Body>
   };
 
   const surveyTask = await client.taskrouter.workspaces(context.TWILIO_WORKSPACE_SID).tasks.create({
-    workflowSid: 'WW6a967d8f663083bdb8ae586539fa71d7', // TODO: move this out
```

Comment:
While this and the values on Line 73 aren't really secrets, it would still be preferable to keep them out of a public repo completely.